src/windows/MediaProxy.js

```diff
@@ -33,8 +42,11 @@ module.exports = {
     create:function(win, lose, args) {
         var id = args[0];
         var src = args[1];
+        var createAudioNode = args[2];
```

Comment:
This is later compared to true using ===.  Suggest coercing it to true or false here, "var createAudioNode = !!args[2];".